cmd/restic/lock_test.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"context"
+	"github.com/restic/restic/internal/repository"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/restic/restic/internal/test"
+)
+
+func TestLockWaitTimeout(t *testing.T) {
+	repo, cleanup := repository.TestRepository(t)
+	defer cleanup()
+
+	elock, err := lockRepoExclusive(context.TODO(), repo.(*repository.Repository))
+	test.OK(t, err)
+
+	wait := 2 * time.Second
```

Comment:
Would it be possible to reduce the wait duration further? This test and the next add nearly 4 seconds to the unit test execution.